process-automation/latest/model/interfaces.md

```diff
@@ -36,6 +36,54 @@ To create a user interface:
 
 4. Enter a name and optional description.
 
+## Theme
+
+You can add a theme to the Digital Workspace. There are several different ways of doing this and how you do it will depend on the type of user you are.
```

Comment:
Now there is only one way :) i.e. using modelin app

Reply:
Removed the entire sentence.